packages/dd-trace/test/ci-visibility/exporters/ci-visibility-exporter.spec.js

```diff
@@ -90,20 +90,22 @@ describe('CI Visibility Exporter', () => {
         .reply(202, '')
 
       const ciVisibilityExporter = new CiVisibilityExporter({ port, isGitUploadEnabled: true })
-      ciVisibilityExporter.getItrConfiguration({}, () => {
-        expect(scope.isDone()).not.to.be.true
+      ciVisibilityExporter._resolveCanUseCiVisProtocol(true)
+      ciVisibilityExporter.getLibraryConfiguration({}, () => {})
+      ciVisibilityExporter._gitUploadPromise.then(() => {
+        expect(scope.isDone()).to.be.true
         done()
       })
```

Comment:
only change in the PR that I can't tell if it's correct or not. Just asking for your attention here just in case

Reply:
yeah good catch. This was actually a bug: it said in the test name `should upload git metadata` but then we were asserting `expect(scope.isDone()).not.to.be.true`, which is to say we haven't uploaded anything. 

The reason why `scope.isDone()` was false was because we were doing the assertion before `_gitUploadPromise` was resolved 

I should've added a note 😄